beacon-chain/db/filesystem/blob.go

```diff
@@ -71,14 +63,16 @@ func NewBlobStorage(base string, opts ...BlobStorageOption) (*BlobStorage, error
 			return nil, fmt.Errorf("failed to create blob storage at %s: %w", base, err)
 		}
 	}
+	if b.pruner == nil {
+		log.Warn("Initializing blob filesystem storage with pruning disabled")
```

Comment:
since we're using func opts I thought it was a good idea to complain when there is no pruner.